src/pages/patientView/timeline/styles.scss

```diff
@@ -10,9 +10,10 @@
     font-size: 10px;
 }
 
-.timeline-label {
-    font-family: sans-serif;
-    font-size: 12px;
+#timeline > svg:first-child {
+    .timeline-label:last-of-type {
```

Comment:
@inodb that label seems to be reliably the last child element (even though it appears first).  this is pretty icky but ...